project/profiles/capacity4more/modules/c4m/og/c4m_og/c4m_og.module

```diff
@@ -1631,7 +1631,17 @@ function c4m_og_get_user_administered_groups_count($uid, $node_type) {
  *   Notification types.
  */
 function c4m_og_get_group_content_notification_types() {
-  return array('discussion', 'document', 'event', 'wiki_page');
+  return array('discussion', 'document', 'event', 'wiki_page', 'news');
```

Comment:
@anvmn 
be aware that the `news` content type is only for projects and `discussion` is only for groups.